src/classes/Cart.ts

```diff
@@ -231,7 +231,11 @@ abstract class Cart {
 
         this.offer.data('handleTimestamp', moment().valueOf());
 
-        this.offer.setMessage('Powered by TF2 Automatic');
+        if (!process.env.OFFER_MESSAGE) {
```

Comment:
Change it to just one line. offer.setMessage(‘Powered by TF2 Automatic’ + (process.env.OFFER_MESSAGE ? (‘.’ + process.env.OFFER_MESSAGE) : ‘’));

Reply:
You don’t need to change it to exactly what I say, I don’t have error checks in the browser like you do in the editor. Please fix the errors before you commit.

Reply:
oh god.

Reply:
Just read the errors, they are telling you what to fix.

Reply:
You’re not doing what I am telling you to do. Install the prettier extention. Also read the contributing guide so you don’t have problems like with this pull request: https://github.com/Nicklason/tf2-automatic/blob/master/.github/contributing.md#pull-requests

Reply:
I'm editing it directly from the browser.

Reply:
WHY would you do that when I am telling you to set up prettier and eslint?

Reply:
ok sorry.

Reply:
I'm passing this to anyone.

Reply:
It’s fine just do what I am telling you to do instead of making the same mistakes over and over again.

Reply:
> I'm passing this to anyone.

You can just make a new pull request and fix the issues. You just need to use your editor with prettier and eslint installed.